strings-csharp/StringFormattableString/StringFormattableStringTests/FormattableStringsUnitTests.cs

```diff
@@ -0,0 +1,53 @@
+using StringFormattableString;
+using System.Runtime.CompilerServices;
+
+namespace StringFormattableStringTests
+{
+    [TestClass]
+    public class FormattableStringsUnitTests
+    {
+        private readonly FormattableStringsMethods _formattableStringMethods;
+        private readonly string _sampleString;
+        private readonly FormattableString _sampleFormattableString;
```

Comment:
As these are only used in a single method each, we don't need to define then as private fields. This ties into a comment I left on the feedback doc. If we move these into the methods, it will make things clearer in the article.